i18n/hourofcode.com/out.rb

```diff
@@ -1,13 +1,126 @@
+  'Khmer' => ['km-KH', 'km'],
+  'Korean' => ['ko-KR', 'ko'],
+  'Kurdish' => ['ku-IQ', 'ku'],
+  'Latvian' => ['lv-LV', 'lv'],
+  'Lithuanian' => ['lt-LT', 'lt'],
+  'Macedonian (FYROM)' => ['mk-MK', 'mk'],
+  'Malay' => ['ms-MY', 'ms'],
+  'Maltese' => ['mt-MT', 'mt'],
+  'Marathi' => ['mr-IN', 'mr'],
+  'Nepali' => ['ne-NP', 'ne'],
+  'Norwegian' => ['no-NO', 'no'],
+  'Norwegian Nynorsk' => ['nn-NO', 'nn'],
+  'Pashto' => ['ps-AF', 'ps'],
+  'Persian' => ['fa-IR', 'fa'],
+  'Polish' => ['pl-PL', 'pl'],
+  'Portuguese' => ['pt-PT', 'po', 'pt'],
+  'Portuguese, Brazilian' => ['pt-BR', 'pt'],
+  'Romanian' => ['ro-RO', 'ro'],
+  'Russian' => ['ru-RU', 'ru'],
+  'Serbian (Cyrillic)' => ['sr-SP', 'sr'],
+  'Slovak' => ['sk-SK', 'sk'],
+  'Slovenian' => ['sl-SI', 'sl'],
+  'Spanish' => ['es-ES', 'es'],
+  'Spanish, Mexico' => ['es-MX', 'la', 'es-MX'],
+  'Swedish' => ['sv-SE', 'sv'],
+  'Tamil' => ['ta-IN', 'ta'],
+  'Thai' => ['th-TH', 'th'],
+  'Turkish' => ['tr-TR', 'tr'],
+  'Ukrainian' => ['uk-UA', 'uk'],
+  'Urdu (Pakistan)' => ['ur-PK', 'ur'],
+  'Vietnamese' => ['vi-VN', 'vi']
+}
+
+
+#########################################################################################
+##                                                                                     ##
+## rename folders and files from crowdin codes to our codes                            ##
+##                                                                                     ##
+#########################################################################################
+
+languages.each_pair do |name, codes|
+  # rename downloaded folders from language to locale
+  if File.directory?("../locales/#{name}/")
+    FileUtils.cp_r "../locales/#{name}/.", "../locales/#{codes[0]}"
+    FileUtils.rm_r "../locales/#{name}"
+  end
+
+  # rename yml file from en.yml to code
+  File.rename("../locales/#{codes[0]}/hourofcode/en.yml", "../locales/#{codes[0]}/hourofcode/#{codes[1]}.yml")
+
+  # edit the code in the yml file for special coded languages
+  unless codes[2].nil?
+  	file = "../locales/#{codes[0]}/hourofcode/#{codes[1]}.yml"
+		File.write(file, File.read(file).gsub(/#{codes[2]}:/, "#{codes[1]}:"))
+  end
 end
 
+
+
+#########################################################################################
+##                                                                                     ##
+## copy files from i18n to hoc.com                                                     ##
+##                                                                                     ##
+#########################################################################################
+
+languages.each_pair do |name, codes|
+	i18n_dir = Dir["../locales/#{codes[0]}/hourofcode"]
+	hoc_dir = Dir["../../pegasus/sites.v3/hourofcode.com/i18n"]
+
+	FileUtils.cp(i18n_dir + "#{codes[1]}.yml", hoc_dir)
+
+	i18n_dir.each do |file|
+		FileUtils.cp(i18n_dir + "/**/*.md", hoc_dir + "/public/#{codes[1]}")
```

Comment:
Style/IndentationWidth: Use 2 (not 1) spaces for indentation.<br>Style/Tab: Tab detected.